Softeq.XToolkit.ProjectInfrastructure/HttpRequest.cs

```diff
@@ -49,6 +53,34 @@ public HttpRequest(Uri uri, HttpMethods method) : this(uri)
             CustomHeaders = new Dictionary<string, object>(headers ?? new Dictionary<string, object>());
         }
 
+        public object Data
+        {
+            get => _data;
+            set
```

Comment:
Do we really need a public setter here? If yes - then `_serializedData = null;` part should be also using _serializeDataLock